src/components/SelectedFolderRender.js

```diff
@@ -0,0 +1,45 @@
+import { getTimeAgo } from '../getTimeAgo';
+import FloatingAddBtn from './FloatingAddBtn';
+import './FloatingAddBtn.css';
+
+function formatDate(value) {
+  const date = new Date(value);
+  return `${date.getFullYear()}. ${date.getMonth() + 1}. ${date.getDate()}`;
+}
+
+function SelectedFolderRender({ selectedFolder }) {
+  return (
+    <ul className="folder-card">
+      {selectedFolder?.map(link => (

```

Comment:
map 안에서는 명확하게 link로 파라미터네이밍 해주셨네요 👍  배열이름도 개선하시면 좋겠습니다.
```suggestion
      {links?.map(link => (

```